packages/manager/modules/nasha/src/dashboard/partition/snapshots/translations/Messages_fr_FR.json

```diff
@@ -15,5 +15,7 @@
   "nasha_dashboard_partition_snapshots_reset_snapshot_types": "Supprimer les changements de fréquences",
   "nasha_dashboard_partition_snapshots_create_rule": "Sont autorisés : caractères alphanumériques, point, deux points et tiret",
   "nasha_dashboard_partition_snapshots_warning_1": "20% d'espace additionnel vous est fourni pour vos snapshots.",
-  "nasha_dashboard_partition_snapshots_warning_2": "Dans le cas où vous dépasseriez cet espace, les snapshots seront stockés sur votre espace de stockage primaire."
+  "nasha_dashboard_partition_snapshots_warning_2": "Dans le cas où vous dépasseriez cet espace, les snapshots seront stockés sur votre espace de stockage primaire.",
+  "nasha_dashboard_partition_snapshots_count_title": "Nombre de snapshots",
+  "nasha_dashboard_partition_snapshots_count": "{{ count }}/{{ max }} (maximum)"
```

Comment:
Suggestion: Can we move the calculation {{ count }}/{{ max }}to controller ?
"nasha_dashboard_partition_snapshots_count": "{{ maximum }} (maximum)"

Reply:
This is not a calculation but a display of information. Would you prefer I use this.$translate.instant()?
![count_snapshots](https://user-images.githubusercontent.com/67371070/185063352-124c378d-f133-4d8e-a39f-40c50bdb7389.png)

Reply:
Okay Guillaume you can ignore my comments

Reply:
@ghyenne , it is better if you maintain only `(maximum)` in the trans file.

Reply:
I created a variable in controller:

```
this.snapshotsCount = `${
      this.customSnapshots.length
    }/${MAX_CUSTOM_SNAPSHOT} (${this.$translate.instant(
      'nasha_dashboard_partition_snapshots_count_max',
    )})`;
 ```